src/Sulu/Component/Content/Types/BlockContentType.php

```diff
@@ -328,10 +354,22 @@ private function prepareData(PropertyInterface $property, callable $dataCallback
         $data = [];
         for ($i = 0; $i < $blockProperty->getLength(); ++$i) {
             $blockPropertyType = $blockProperty->getProperties($i);
+            $blockPropertyTypeSettings = $blockPropertyType->getSettings();
+
+            if (
```

Comment:
Not exactly, the `is_array` call is still needed, because a `stdClass` might be passed. In that case you get the following error: 

```
Cannot use object of type stdClass as array
```

But I can use `empty` instead of `isset`

Reply:
Ok then its fine for me this way.